internal/server/openapi.go

```diff
@@ -217,10 +217,6 @@ func WriteOpenAPIDocToFile(openAPIDoc openapi3.T, version string, filename strin
 }
 
 func setTagInfo(f reflect.StructField, t, parent reflect.Type, schema, parentSchema *openapi3.Schema) {
-	if ex := getDefaultExampleForType(t); len(ex) > 0 {
-		schema.Example = ex
-	}
```

Comment:
This call wasn't necessary because we already set these defaults in `setTypeInfo`, which is always called in exactly the same place as `setTagInfo`, so I was able to remove this.